test/views/declaration/summary/SummaryPageViewAmendSpec.scala

```diff
@@ -56,5 +56,7 @@ class SummaryPageViewAmmendSpec extends SummaryPageViewSpec {
       backButton must haveHref(controllers.routes.SubmissionsController.displayListOfSubmissions())
     }
 
+    "display Save and Come back later link" in {}
+
```

Comment:
Empty test????

Reply:
Whoops, I left this in after adding the test to `commonBehaviour` method in `test/views/declaration/summary/SummaryPageViewSpec.scala`